packages/apps/docs/src/components/Layout/__fixtures__/leftMenuTree.ts

```diff
@@ -191,9 +191,9 @@ export const fullLayoutLeftMenuTree: IMenuItem[] = [
       {
         children: [],
         root: '/kadena/client-examples',
-        title: 'Client examples',
-        menu: 'Client examples',
-        label: 'Client examples',
+        title: 'Client Examples',
+        menu: 'Client Examples',
+        label: 'Client Examples',
```

Comment:
@sstraatemans I changed these to make sure the metadata in the markdown used the right capital letters)